File: libRxTask/src/main/java/com/ypz/killetom/librxtask/exception/RxTaskCancelException.java
package com.ypz.killetom.librxtask.exception;

public class RxTaskCancelException extends RxTaskSuperException {

    public RxTaskCancelException() {
        super();
    }

    public RxTaskCancelException(String message) {
        super(message);
    }

    public RxTaskCancelException(String message, Throwable cause) {
        super(message, cause);
    }

    public RxTaskCancelException(Throwable cause) {
        super(cause);
    }
}

File: libRxTask/src/main/java/com/ypz/killetom/librxtask/exception/RxTaskCancelException.kt
package com.ypz.killetom.librxtask.exception

/**
 * @ProjectName: RxTask
 * @Package: com.ypz.killetom.librxtask.exception
 * @ClassName: RxTaskCancelException
 * @Description: task 取消异常
 * @Author: KilleTom
 * @CreateDate: 2020/4/26 17:30
 * @UpdateUser: 更新者
 * @UpdateDate: 2020/5/18 12:00
 * @UpdateRemark: 创建
 * @Version: 1.0
 */
class RxTaskCancelException : RxTaskSuperException {
    constructor() : super() {}
    constructor(message: String?) : super(message) {}
    constructor(message: String?, cause: Throwable?) : super(message, cause) {}
    constructor(cause: Throwable?) : super(cause) {}
}